### PR TITLE
Add Plaid Account Token to Tokens API

### PR DESCRIPTION
**Pull request by slavcodev**
image

### CODE REVIEW

**Comment by slavcodev** [comment]
php please review

File: openapi/components/schemas/PaymentTokens/PaymentToken.yaml [hunk 1]
```diff
@@ -2,6 +2,7 @@ type: object
 title: Payment Token
 allOf:
   - $ref: ../Common/CommonPaymentToken.yaml
+  - $ref: ./PaymentTokenDeprecatedProperties.yaml
```
Comment (adamaltman) [inline openapi/components/schemas/PaymentTokens/PaymentToken.yaml:5]: I would remove all of these... I think this is a bad addition.

**Review by adamaltman — DISMISSED** [review]
I don't like adding deprecated properties as a separate file, and I believe we intentionally removed it from the documentation before.

**Comment by slavcodev** [comment]
We didn't remove them from docs before. Now it is commen fields, I've removed it in this PR 
https://github.com/Rebilly/api-definitions/pull/166/files#diff-70113a5a1e0f61a2a6e506a45c42bf01L17

I segregate it in new file to not add to new token types like we did for `DififtalWalletToken`.
Let me know if we need to remove it from all tokens, I'm 👍  for that

File: openapi/components/schemas/PaymentTokens/PaymentToken.yaml (hunk 1 above)
Comment (slavcodev) [reply openapi/components/schemas/PaymentTokens/PaymentToken.yaml:5]: It's not an additional, just to make clear. See `CommonPaymentToken`
 https://github.com/Rebilly/api-definitions/pull/166/files#diff-70113a5a1e0f61a2a6e506a45c42bf01L17

**Comment by adamaltman** [comment]
Let's remove it from all tokens (the backend will continue to support it). 

It's been marked as deprecated for a while, and nobody should get an idea that they should try it now.

**Review by ashkarpetin — DISMISSED** [review]

**Review by subenson — DISMISSED** [review]

**Comment by slavcodev** [comment]
php please review

**Review by ashkarpetin — APPROVED** [review]

**Review by wmdmgv — APPROVED** [review]

**Pull request merged**